pkg/plugins/golang/declarative/v1/init.go

```diff
@@ -41,36 +41,46 @@ func (p *initSubcommand) InjectConfig(c config.Config) error {
 }
 
 func (p *initSubcommand) Scaffold(fs machinery.Filesystem) error {
-	err := updateDockerfile()
-	if err != nil {
-		return err
-	}
-	return nil
+	return updateDockerfile()
 }
 
 // updateDockerfile will add channels staging required for declarative plugin
 func updateDockerfile() error {
 	fmt.Println("updating Dockerfile to add channels/ directory in the image")
-	managerFile := filepath.Join("Dockerfile")
+	dokerfile := filepath.Join("Dockerfile")
```

Comment:
nits: maybe `dockerfile`?

Reply:
Because it was wrong. The file update is the Dockerfile and not manager file. 
Could to be in another PR.